engine/src/main/java/org/terasology/world/block/BlockRegion.java

```diff
@@ -25,20 +25,39 @@
  */
 public class BlockRegion {
 
+
+    /**
+     * The x coordinate of the minimum corner.
+     */
+    public int minX = Integer.MAX_VALUE;
+    /**
+     * The y coordinate of the minimum corner.
+     */
+    public int minY = Integer.MAX_VALUE;
+    /**
+     * The z coordinate of the minimum corner.
+     */
+    public int minZ = Integer.MAX_VALUE;
+    /**
+     * The x coordinate of the maximum corner.
+     */
+    public int maxX = Integer.MIN_VALUE;
+    /**
+     * The y coordinate of the maximum corner.
+     */
+    public int maxY = Integer.MIN_VALUE;
     /**
-     * AABB region that backs a BlockRegion
+     * The z coordinate of the maximum corner.
      */
-    public final AABBi aabb = new AABBi();
+    public int maxZ = Integer.MIN_VALUE;
```

Comment:
@pollend Should those really be public? I'm a bit confused right now, as I thought we agreed on Slack with @4Denthusiast that we should ensure that invalid min/max combinations should throw exceptions. With these values publicly accessible we cannot guarantee that...